src/server/lock_manager/deadlock.rs

```diff
@@ -60,6 +60,9 @@ struct Locks {
     last_detect_time: Instant,
 }
 
+const MAX_SAMPLE_INTERVAL: Duration = Duration::from_secs(15);
```

Comment:
Please add comments for the constant variables.

Reply:
Updated in the description